ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/rlpx/wire/WireMessagesSedesTest.java

```diff
@@ -59,6 +67,8 @@ private static void assertSedesWorks(final byte[] data) {
     // Re-serialize and check that data matches
     final BytesValueRLPOutput out = new BytesValueRLPOutput();
     peerInfo.writeTo(out);
-    assertThat(out.encoded()).isEqualTo(input);
+    if (encEqualsInput) {
+      assertThat(out.encoded()).isEqualTo(input);
+    }
```

Comment:
this is necessary because we don't encode zero in the same way? Can we link to the ticket that will fix this?

Reply:
I can add a comment!